hail/src/main/scala/is/hail/methods/MatrixExportColsParallel.scala

```diff
@@ -0,0 +1,143 @@
+package is.hail.methods
+
+import java.io.OutputStreamWriter
+
+import is.hail.annotations.{UnsafeIndexedSeq, UnsafeRow}
+import is.hail.expr.TableAnnotationImpex
+import is.hail.expr.ir.MatrixValue
+import is.hail.expr.ir.functions.MatrixToValueFunction
+import is.hail.expr.types.MatrixType
+import is.hail.expr.types.virtual.{TVoid, Type}
+import is.hail.utils._
+import org.apache.spark.TaskContext
+import org.apache.spark.sql.Row
+
+case class MatrixExportColsParallel(parallelism: Int, path: String, bgzip: Boolean) extends MatrixToValueFunction {
+  def typ(childType: MatrixType): Type = TVoid
+
+  def execute(mv: MatrixValue): Any = {
+
+    val hConf = mv.sparkContext.hadoopConfiguration
+
+    hConf.delete(path, recursive = true) // overwrite by default
```

Comment:
Should we include an overwrite flag exposed to the user?

Reply:
I don't want to spend the time to add that right now - maybe if it's actually used and promoted to a non-experimental method? :)

Reply:
ok.